third_party/proto/cosmos/base/query/v1beta1/pagination.proto

```diff
@@ -30,6 +30,9 @@ message PageRequest {
   // count_total is only respected when offset is used. It is ignored when key
   // is set.
   bool count_total = 4;
+
+  // reverse is set to true if results are to be returned in the descending order.
+  bool reverse = 5;
```

Comment:
This needs to be changed upstream in the SDK. We're just copying proto definitions here in `third_party`

Reply:
nevermind, looks like this is just updating the files

Reply:
This came in from the `make proto-update-deps` command, so being copied from upstream